calibrate.py

```diff
@@ -16,6 +16,40 @@
 V_RMS = 220
 I_RMS = 10
 
+def ask_for_circuits_to_calibrate():
```

Comment:
I would replace this interactive asking with command line arguments using [argparse](https://docs.python.org/2/library/argparse.html#module-argparse). Something like:
```
$ python calibrate.py -c 1,2,4
$ python calibrate.py --all (default)
$ python calibrate.py -c 5
```